packages/nimble-components/src/patterns/dropdown/styles.ts

```diff
@@ -187,55 +227,33 @@ export const styles = css`
         border-top-right-radius: 0;
     }
 
-    :host([open][position='above']) .anchored-region {
-        padding-bottom: ${smallPadding};
+    .scrollable-region {
+        overflow-y: auto;
     }
 
-    :host([open][position='below']) .anchored-region {
-        padding-top: ${smallPadding};
+    .listbox slot {
+        display: block;
+        background: transparent;
+        padding: var(--ni-private-listbox-padding);
     }
 
-    .selected-value {
-        flex: auto;
-        font-family: inherit;
-        text-align: start;
-        white-space: nowrap;
-        text-overflow: ellipsis;
-        overflow: hidden;
-        padding: 0px;
-        padding-left: ${mediumPadding};
+    .empty slot {
```

Comment:
pretty generic class name, including the class for the element makes it clearer


```suggestion
    .listbox.empty slot {
```